str/gangstr_runner.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+"""
+Example of running Batch script with analysis-runner.
+"""
+import os
+import hailtop.batch as hb
+import click
+
+DATASET = os.getenv('DATASET')
+HAIL_BUCKET = os.getenv('HAIL_BUCKET')
+OUTPUT_SUFFIX = os.getenv('OUTPUT')
+BILLING_PROJECT = os.getenv('HAIL_BILLING_PROJECT')
+ACCESS_LEVEL = os.getenv('ACCESS_LEVEL')
+
+REF_FASTA = 'gs://cpg-reference/hg38/v1/Homo_sapiens_assembly38.fasta'
+SAMTOOLS_IMAGE = 'australia-southeast1-docker.pkg.dev/cpg-common/images/samtools:v0'
+GANGSTR_IMAGE = "australia-southeast1-docker.pkg.dev/cpg-common/images/gangstr:v2.5"
+regions = 'gs://cpg-fewgenomes-test/hoptan-str/gangstr_catalog_with_offtarget.bed'
+
+
+@click.command()
+@click.argument('cram_path')
+@click.argument('region')
+def main(cram_path: str, region: str):  # pylint: disable=missing-function-docstring
+    """
+    Subset CRAM or BAM file CRAM_PATH to REGION. Example: batch.py sample.cram chr21:1-10000
+    """
+    # Initializing Batch
+    backend = hb.ServiceBackend(billing_project=BILLING_PROJECT, bucket=HAIL_BUCKET)
+    b = hb.Batch(backend=backend, default_image=os.getenv('DRIVER_IMAGE'))
+
+    # Adding a job and giving it a descriptive name.
+    j = b.new_job('Subset CRAM')
+
+    # Making sure Hail Batch would localize both CRAM and the correponding CRAI index
+    cram = b.read_input_group(**{'cram': cram_path, 'cram.crai': cram_path + '.crai'})
+
+    # Working with CRAM files requires the reference fasta
+    ref = b.read_input_group(
+        **dict(
+            base=REF_FASTA,
+            fai=REF_FASTA + '.fai',
+            dict=REF_FASTA.replace('.fasta', '').replace('.fna', '').replace('.fa', '')
+            + '.dict',
+        )
+    )
+
+    # This image contains basic bioinformatics tools like samtools, bcftools, Picard, etc.
+    j.image(SAMTOOLS_IMAGE)
+    j.image(GANGSTR_IMAGE)
+
+    # For larger CRAMs, request more storage.
+    j.storage('10G')
+
+    # If you want to run a multithreaded command, e.g. samtools with -@, request more CPUs here.
+    # Note that the machines use hyperthreading, so for every CPU, 2x threads are available.
+    j.cpu(2)
+
+    # The command that do the actual job.
+    j.command(
+        f"""
+
+    samtools view -T {ref.base} {cram['cram']} {region} -Obam -o {j.output_bam}    
+    samtools index {j.output_bam}
+    gangstr --bam {j.output_bam} --ref ref.base --regions {regions} --out {j.gangstr_trial}
+    """
+    )
+
+    # Speciying where to write the result
+    out_fname = os.path.splitext(os.path.basename(cram_path))[0] + '-split.gang'
+    output_path = f'gs://{HAIL_BUCKET}/{OUTPUT_SUFFIX}/{out_fname}'
+    b.write_output(j.gangstr_trial, output_path)
+
+    # don't wait for the hail batch workflow to complete, otherwise
+    # the workflow might get resubmitted if this VM gets preempted.
+    b.run(wait=False)
+
+
+if __name__ == '__main__':
+    main()  # pylint: disable=no-value-for-parameter
```

Comment:
We like a newline character at the end of files :)
```suggestion
    main()  # pylint: disable=no-value-for-parameter
    
```